fdbmonitor/fdbmonitor.cpp

```diff
@@ -511,7 +545,11 @@ struct Command {
 			if (!strcmp(i.pItem, "command") || !strcmp(i.pItem, "restart-delay") ||
 			    !strcmp(i.pItem, "initial-restart-delay") || !strcmp(i.pItem, "restart-backoff") ||
 			    !strcmp(i.pItem, "restart-delay-reset-interval") || !strcmp(i.pItem, "disable-lifecycle-logging") ||
-			    !strcmp(i.pItem, "delete-envvars") || !strcmp(i.pItem, "kill-on-configuration-change")) {
+			    !strcmp(i.pItem, "delete-envvars") || !strcmp(i.pItem, "kill-on-configuration-change") ||
+			    !strcmp(i.pItem, "restart_delay") || !strcmp(i.pItem, "initial_restart_delay") ||
+			    !strcmp(i.pItem, "restart_backoff") || !strcmp(i.pItem, "restart_delay_reset_interval") ||
+			    !strcmp(i.pItem, "disable_lifecycle_logging") || !strcmp(i.pItem, "delete_envvars") ||
+			    !strcmp(i.pItem, "kill_on_configuration_change")) {
```

Comment:
You could also optionally hide this logic behind a function call, which would both keep this simpler and make it easier to avoid issues with future flags.